src/services/anchor-service.ts

```diff
@@ -305,10 +305,11 @@ export default class AnchorService {
 
     let request = null;
     for (let index = 0; index < requests.length; index++) {
+      let docId
       try {
         request = requests[index];
 
-        const docId = this._getRequestDocID(request)
+        docId = this._getRequestDocID(request)
```

Comment:
Why the change to let here?

Reply:
I guess because `docId` is used in the catch scope.